packages/hub/indexing/card-utils.ts

```diff
@@ -113,12 +126,20 @@ async function generateCardModule(card: SingleResourceDoc) {
     data: { ...card.data },
```

Comment:
RE the TODO comment above: 

> TODO it's possible for cards originating from the same package to have different templates/components as a specific card instance could have its schema altered.

I think this isn't the right way to think about it. Every card, including every card instance, *is* a package. Two card instances that happen to have the same exact contents other than their individual card data can share one on-disk package for holding their code, *but this is only an optimization*.  In principle, every card instance is a complete package.

Reply:
ok, I'll refactor this to organize the on disk artifacts by card id instead.